packages/camera/README.md

```diff
@@ -15,13 +15,18 @@ First, add `camera` as a [dependency in your pubspec.yaml file](https://flutter.
 
 ### iOS
 
-Add a row to the `ios/Runner/Info.plist` of your app with the key `Privacy - Camera Usage Description` and a usage description.
+Add two rows to the `ios/Runner/Info.plist`:
+
+* one with the key `Privacy - Camera Usage Description` and a usage description.
+* and one with the key `Privacy - Microphone Usage Description` and a usage description.
```

Comment:
@holykou @nisdis can you confirm this small change I made? do we actually need to force the usage of the mic? My thinking is that camera is also use for picture where mic is not used. Btw I haven't tested but what happens if user doesn't allow the mic, can we continue with just the video?

Reply:
The microphone permission is requested only for video, not for clicking pictures. I'm not adding audio input when the camera is started, the user can click pictures without having to set microphone pictures. I haven't tested what happens when the permission is denied for the microphone either.

Reply:
Okay here there could be an issue. When I tried running the app for the first time. It asked for permission video and then on start of the video capture it asked for microphone permission (after the plist fix). 
The issue is when you try to capture the second time the app crashes as it re-add the microphone input.
I added the microphone input on camera start to solve this issue.
 But we could re-think this and call it only when required.